frontend/src/app_context.tsx

```diff
@@ -7,8 +7,20 @@ import {
 } from "react";
 import { LoginState } from "./constants";
 
+export enum WordVaultFontStyle {
+  Monospace = "monospace",
+  SansSerif = "sans-serif",
+  Tiles = "tiles",
+}
+
 type DisplaySettings = {
-  fontStyle: string;
+  /**
```

Comment:
Feel free to correct (or remove) these comments, was just me making sure I kept the settings straight